python/Ganga/test/GPI/TestSJSubmit.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+class TestSJSubmit(GangaUnitTest):
+
+    n_subjobs = 10
+
+    def setUp(self):
+        """Make sure that the Job object isn't destroyed between tests"""
+        super(TestSJSubmit, self).setUp()
+        from Ganga.Utility.Config import setConfigOption
+        setConfigOption('Output', 'FailJobIfNoOutputMatched', 'True')
+        setConfigOption('TestingFramework', 'AutoCleanup', 'False')
+        setConfigOption('Configuration', 'resubmitOnlyFailedSubjobs', 'True')
+
+    def _getSplitter(self):
```

Comment:
This method doesn't use `self` so could probably be a `@staticmethod` or even a free function.